Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -0,0 +1,144 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import asyncio
+
+import octobot_commons.symbol_util as symbol_util
+
+import octobot_trading.modes as trading_modes
+import octobot_trading.enums as trading_enums
+import octobot_trading.constants as trading_constants
+import octobot_trading.util as trading_util
+import octobot_trading.errors as trading_errors
+import octobot_trading.personal_data as trading_personal_data
+
+
+class DCATradingModeConsumer(trading_modes.AbstractTradingModeConsumer):
+    AMOUNT_TO_BUY_IN_REF_MARKET = "amount_to_buy_in_reference_market"
+    ORDER_PRICE_DISTANCE = 0.005
```

Comment:
0.5% ? I think something like 0.1% would be better to unsure it's taken as maker but is not missed

Reply:
That's also right :+1: